lib/gui/app.js

```diff
@@ -154,6 +155,18 @@ app.run(() => {
         allowSleepUpdateCheck: currentReleaseType === release.RELEASE_TYPE.PRODUCTION
       });
     });
+
+  // If the error is an update user error, then we don't want
+  // to bother users each time they open the app.
+  // See: https://github.com/resin-io/etcher/issues/1525
+  }).catch({
```

Comment:
Shouldn't we first check that we've got a userError, and _then_ check the error's `code` ?